docs/sce_file_contents.md

```diff
@@ -296,3 +303,161 @@ For methods that rely on the HTO data, these statistics are found in the `colDat
 
 Genetic demultiplexing statistics are found in the main `colData(sce)` data frame, with the prefix `vireo_`.
 
+## Components of an `AnnData` object
+
+Before getting started, we highly encourage you to familiarize yourself with the general `AnnData` object structure and functions available as part of the [`AnnData` package](https://anndata.readthedocs.io/en/latest/index.html).
+For the most part, the `AnnData` objects that we provide are formatted to match the expected data format for [`CELLxGENE`](https://cellxgene.cziscience.com/) following [schema version `3.0.0`](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md).
+
+To begin, you will need to load the `AnnData` package and read the HDF5 file:
+
+```python
+import anndata
+adata_object = anndata.read_h5ad("SCPCL000000_processed_rna.hdf5")
+```
+
+### `AnnData` expression counts
+
+The data matrix, `X`, of the `AnnData` object for single-cell and single-nuclei experiments contains the primary RNA-seq expression data as integer counts in both the unfiltered (`_unfiltered_rna.hdf5`) and filtered (`_filtered_rna.hdf5`) objects.
+The data is stored as a sparse matrix, and each column represents a cell or droplet, each row a gene.
+Column names are cell barcode sequences and row names are Ensembl gene IDs.
+The `X` matrix can be accessed with the following python code:
+
+```python
+raw_count_matrix = adata_obj.X
```

Comment:
I think I might mark the values here and below as comments and leave off the assignment.
```suggestion
adata_obj.X # raw count matrix
```